python/ray/cloudpickle/cloudpickle_fast.py

```diff
@@ -541,8 +541,8 @@ def reducer_override(self, obj):
         # This is a patch for python3.5
         if isinstance(obj, numpy.ndarray):
             if (self.proto < 5 or
-                    (not obj.flags.c_contiguous and
-                     not obj.flags.f_contiguous) or
+                    (not obj.flags.c_contiguous and not obj.flags.f_contiguous) or
```

Comment:
@suquark are these fixes being upstreamed to cloudpickle?

Reply:
There has been dramatic changes to the upstream (e.g. drop of python2 support). We definitely need to sync with them first. I'll create a PR later to sync with the upstream, then we could try to update changes to the upstream.